torchao/prototype/parq/quant/uniform_torchao.py

```diff
@@ -50,8 +50,23 @@ def __init__(
         self.quant_min = quant_min
         self.quant_max = quant_max
         self.eps = eps
-        self.preserve_zero = preserve_zero
-        self.zero_point_domain = zero_point_domain
+
+        # defaults: zero_point_domain=ZeroPointDomain.INT, preserve_zero=True
+        self._choose_qparams = choose_qparams_affine
+        self._quantize = quantize_affine
+        self._dequantize = dequantize_affine
+
+        if zero_point_domain == ZeroPointDomain.FLOAT and not preserve_zero:
+            self._choose_qparams = choose_qparams_affine_tinygemm
+            self._quantize = quantize_affine_tinygemm
+            self._dequantize = dequantize_affine_tinygemm
+        elif zero_point_domain == ZeroPointDomain.INT and not preserve_zero:
+            self._choose_qparams = choose_qparams_affine_dont_preserve_zero
+            self._quantize = quantize_affine
+            self._dequantize = dequantize_affine
+        elif zero_point_domain == ZeroPointDomain.NONE:
+            self._quantize = quantize_affine_no_zero_point
+            self._dequantize = dequantize_affine_no_zero_point
```

Comment:
These changes aren't required to make the test pass right, just for clean ups? (we can keep them in this PR, just wanted to ask for my understanding)